vendor/dummy-module.js

```diff
@@ -0,0 +1 @@
+define('qunit-dom', function(){ return {} });
```

Comment:
this is "missing" the `[]` argument from the previous implementation. is that optional?

Reply:
It is, but it does seem silly for me to change this, now that I've gone back to using the same pattern as the old file. I will push another commit so they match exactly.